internal/gatewayapi/testdata/httproute-with-mirror-filter-service-no-port.out.yaml

```diff
@@ -78,10 +78,9 @@ httpRoutes:
         status: "False"
         type: Accepted
       - lastTransitionTime: null
-        message: A valid port number corresponding to a port on the Service must be
-          specified
-        reason: PortNotSpecified
-        status: "False"
+        message: Resolved all the Object references for the Route
+        reason: ResolvedRefs
+        status: "True"
```

Comment:
Should we set ResolvedRefs to false when a mirror filter fails?

Reply:
yes, the original values looked right here